src/cr/cube/cube.py

```diff
@@ -737,28 +778,47 @@ def _flat_values(self):
 
         Mean data may include missing items represented by a dict like
         {'?': -1} in the cube response. These are replaced by np.nan in the
-        returned value. In the case of numeric arrays, we can have list of lists.
-        Returns None if means are not available on the cube.
+        returned value.
         """
         if self._result is None:
             return None
+        return np.array(
+            tuple(np.nan if type(x) is dict else x for x in self._result["data"]),
+            dtype=np.float64,
+        ).flatten()
+
+    @lazyproperty
+    def _result(self):
+        return self._cube_dict.get("result", {}).get("measures", {}).get("mean")
```

Comment:
docstring. Also, this seems a poor name. I would expect `._result` to return `self._cube_dict["result"]`. Maybe something like `._measure_values`.

Reply:
or maybe `._measure_payload`, since it seems it is a dict rather than a list of values.

Reply:
I leave it cause i though it was ok for you after that this property was introduced in the greg's PR 🤷🏽‍♂️  ... I'll change it